openwisp_radius/api/serializers.py

```diff
@@ -128,11 +141,20 @@ def validate(self, data):
             raise serializers.ValidationError(
                 {'number_of_users': _('The field number_of_users cannot be empty')}
             )
-        return super().validate(data)
+        validated_data = super().validate(data)
+        # Additional Model Validation
+        batch_data = data.copy()
+        batch_data.pop('number_of_users', None)
+        slug = batch_data.pop('organization_slug')
+        batch_data.update(organization=Organization.objects.get(slug=slug))
```

Comment:
looks like this raises an exception if the supplied value is not found. Ensure it's resilient by setting it to None if the org is not found so that it raises a validation error (needs an automated test)

Reply:
@nemesisdesign I tried a couple of things, including:

1. Sending non-existing
2. Sending slug of org to which the user doesn't have the access.
3. Sending invalid slug 

In everycase, serializer validation caches the problem even before it reaches the errored line and now that I think about it,
I remember that didn't bother with try/catch for that `batch_data.update(organization=Organization.objects.get(slug=org_slug))` because in normal flow this cannot be empty/incorrect , because `organization_slug` value is checked by serializer validators before this function is called. 

1. Can you please confirm you are indeed in the latest of the PR?
2. Provide information about anything special / different about your instance.
3. Check what your organzation_slug is getting and if that is a valid slug in your database.

Reply:
Since `queryset.filter(pk__in=request.user.organizations_dict.keys())` shows it in the drop down and serializer validators don't raise an error, I am reasonable certain that `organization_slug` should not contain an invalid value.

Reply:
Fixed